site/markdown/contributors/new-developer.md

```diff
@@ -43,10 +43,10 @@ Once we've met you on video call, we can invite you to Slack and add you as a co
 
 ## Step 3: join our biweekly development meetings
 
-We coordinate a lot of our work through meetings. Meetings are on Wednesdays at 20:00 UTC, and we alternate between backend and frontend meetings, so that each runs once every two weeks. Meeting notes are posted on GitHub after the meeting. You can find the calendar invites below:
+We coordinate a lot of our work through meetings. Meetings are on Wednesdays at 20:00 UTC, and we alternate between backend and web meetings, so that each runs once every two weeks. Meeting notes are posted on GitHub after the meeting. You can find the calendar invites below:
 
 * [Backend meeting calendar invite](http://couchers.org/contributors/backend-meeting.ics)
-* [Frontend meeting calendar invite](http://couchers.org/contributors/frontend-meeting.ics)
+* [Frontend meeting calendar invite](http://couchers.org/contributors/web-meeting.ics)
```

Comment:
Should this say web meeting? Or is it one frontend meeting for both web and mobile for the time being?

Reply:
It's separate for now